src/Apps/Auctions/Components/CuritorialRailsTabBar.tsx

```diff
@@ -47,14 +48,22 @@ export const CuritorialRailsTabBarFragmentContainer = createFragmentContainer(
         ...TrendingLotsRail_viewer
         ...StandoutLotsRail_viewer
 
-        me {
-          ...MyBids_me
-          followsAndSaves {
-            artistsConnection {
-              totalCount
+        followedArtistsInAuction: saleArtworksConnection(
+          includeArtworksByFollowedArtists: true
+          isAuction: true
+          liveSale: true
+          first: 1
```

Comment:
non-blocking: `first: 0` fetches metadata without having to return the objects in the list. Maybe it would be good practice to leverage this pattern here to return only the count? 

```graph
saleArtworksConnection(includeArtworksByFollowedArtists: true, first: 0, isAuction: true, liveSale: true) {
		counts {
			total
		}
	}
```

Reply:
Oh that's a great idea and I always forget about this. Will do 👍